src/lib/accounts/superhero.ts

```diff
@@ -13,6 +13,8 @@ import { POPUP_TYPE_MESSAGE_SIGN, POPUP_TYPE_SIGN, POPUP_TYPE_TX_SIGN } from '..
 import { showPopup } from '../../background/popupHandler';
 import type { IPopupType } from '../../types';
 
+// TODO Subhod please check it
+// @ts-ignore
```

Comment:
Please ask Subhod directly to help with solving the problem. Leaving such a comments in the code is a bad thing because it suggests that we are not treating the code in a proper way.